lib/code_corps_web/controllers/github_event_controller.ex

```diff
@@ -19,6 +19,7 @@ defmodule CodeCorpsWeb.GithubEventController do
       github_events =
         GithubEvent
         |> Query.id_filter(params)
+        |> Query.sort_by_inserted_at()
```

Comment:
I wonder if it makes more sense to just call `|> order_by([asc: :inserted_at])` here directly?

Reply:
Technically we would eventually want to wrap this up in a context outside the controller anyway, so it would be more like `GithubEvents.list()` or something in Phoenix 1.3+ style.

Reply:
@JoshSmith Should we do it right now and put a method on GithubEvent?